### PR TITLE
fix UI issues in Profile page layout xml

### PR DESCRIPTION
**Pull request by helmibk1**
Fixes for the first 5 issues for the profile page 
https://docs.google.com/document/d/1TpkA7dxfEdLB2toWleWVP5yu8PgNxmw-WV3ASM7AG0I/edit?pli=1

### CODE REVIEW

File: app/src/main/res/layout/fragment_profile.xml [hunk 1]
```diff
@@ -91,20 +95,19 @@
         app:layout_constraintEnd_toEndOf="parent"
         android:layout_marginTop="30dp" />
 
-    <TextView
+    <androidx.appcompat.widget.AppCompatButton
         android:id="@+id/textDeleteAccount"
         android:layout_width="200dp"
```
Comment (phillipsk) [inline app/src/main/res/layout/fragment_profile.xml:100]: nit: width could be wrap_content

File: app/src/main/res/layout/fragment_profile.xml [hunk 2]
```diff
@@ -64,18 +68,18 @@
...
-        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginTop="52dp"
         app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="20dp"
-        android:layout_marginBottom="40dp"/>
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/txteditprofile" />
 
     <androidx.appcompat.widget.AppCompatButton
```
Comment (phillipsk) [inline app/src/main/res/layout/fragment_profile.xml:84]: nit: width could be wrap_content

**Review by phillipsk — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/card_signin.xml [hunk 3]
```diff
@@ -36,17 +36,17 @@
...
-                    android:textStyle="normal">
+                    android:textStyle="normal"
+                    app:fontFamily="Roboto">
 
                 </TextView>
             </RelativeLayout>
         </com.google.android.material.card.MaterialCardView>
 
-        <!--<com.google.android.material.card.MaterialCardView
+        <com.google.android.material.card.MaterialCardView
```
[the remark this answers is not in the document]
Comment (helmibk1) [reply app/src/main/res/layout/card_signin.xml:49]: This is just a UI fix. For the Facebook functionality, we would need to set up a Facebook developer account.

File: app/src/main/res/layout/fragment_profile.xml (hunk 2 above)
Comment (helmibk1) [reply app/src/main/res/layout/fragment_profile.xml:84]: That will make the width squeezed!

Comment (phillipsk) [reply app/src/main/res/layout/fragment_profile.xml:84]: I tested the changes and it displays the same plus more dynamic

**Review by phillipsk — APPROVED** [review]

**Pull request merged**